### PR TITLE
Update TTS Tasks page.

### PR DESCRIPTION
**Pull request by Vaibhavs10**
Wee update to the models listed, just making it a bit more current.

### CODE REVIEW

File: packages/tasks/src/text-to-speech/about.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ def query(payload):
 	response = requests.post(API_URL, headers=headers, json=payload)
 	return response
 
-output = query({"text_inputs": "This is a test"})
+output = query({"text_inputs": "Max is the best doggo."})
```
Comment (pcuenca) [inline packages/tasks/src/text-to-speech/about.md:28]: 😂

**Review by pcuenca — APPROVED** [review]
Sounds good!

File: packages/tasks/src/text-to-speech/about.md (hunk 1 above)
Comment (Vaibhavs10) [reply packages/tasks/src/text-to-speech/about.md:28]: Facts are facts!

**Comment by Vaibhavs10** [comment]
I forgot what's the protocol here. Is this good to merge?

**Pull request merged**